### PR TITLE
Release v2.11.0

### PR DESCRIPTION
**Pull request by zhiqiangjin**
### Notes:
- Supported Controller version: **UserConnect-5.3.1391** 
- Supported Terraform version: **v0.12.x**

### Features:
1. Implemented support for ``firewall_image_version`` in **aviatrix_firewall_instance**
2. Implemented support for "UDP" ``vpn_protocol`` for AWS ELB-enabled VPN gateways
3. Implemented support for Active-Active HA (``enable_active_active``) in **aviatrix_site2cloud**

### Enhancements:
1. Implemented coverage for ``tag_list`` formatting change due to Boto3
2. Implemented support for attaching TGW VPN connections to different security domains besides the default domain in **aviatrix_aws_tgw_vpn_conn**
3. Implemented cloud_type check to catch incorrect ha_subnet usage for **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
4. Implemented ha_gw_size check to catch incorrect usage when enabling HA for **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**

### Bug Fixes:
1. Fixed issue where ``filtered_spoke_vpc_routes`` caused reordering issues for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**

### CODE REVIEW

**Review by arvindsree — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+## 2.11.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.3.1391** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support of ``firewall_image_version`` parameter into **aviatrix_firewall_instance**
+2. Implemented support of ``vpn_protocol`` for vpn gateway of aws provider with elb enabled in **aviatrix_gateway**
+3. Implemented support of ``enable_active_active`` parameter in **aviatrix_site2cloud**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: 1. Implemented support for ``firewall_image_version`` in **aviatrix_firewall_instance**
2. Implemented support for "UDP" ``vpn_protocol`` for AWS ELB-enabled VPN gateways
3. Implemented support for Active-Active HA (``enable_active_active``) in **aviatrix_site2cloud**

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+### Enhancements:
+1. Implemented coverage for ``tag_list`` formatting change due to Boto3
+2. Implemented support of multiple security domains in **aviatrix_aws_tgw_vpn_conn**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: 2. Implemented support for attaching TGW VPN connections to different security domains besides the default domain

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+3. Implemented check for cloud_type when incorrectly using ha_subnet in **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+4. Implemented check for ``ha_gw_size`` which is only needed for enabling HA in **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+
+### Bug Fixes:
+1. Fixed issue where ``filtered_spoke_vpc_routes`` causes reordering issue in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: 1. Fixed issue where ``filtered_spoke_vpc_routes`` caused reordering issues for **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+### Features:
+1. Implemented support of ``firewall_image_version`` parameter into **aviatrix_firewall_instance**
+2. Implemented support of ``vpn_protocol`` for vpn gateway of aws provider with elb enabled in **aviatrix_gateway**
+3. Implemented support of ``enable_active_active`` parameter in **aviatrix_site2cloud**
+
+### Enhancements:
+1. Implemented coverage for ``tag_list`` formatting change due to Boto3
+2. Implemented support of multiple security domains in **aviatrix_aws_tgw_vpn_conn**
+3. Implemented check for cloud_type when incorrectly using ha_subnet in **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+4. Implemented check for ``ha_gw_size`` which is only needed for enabling HA in **aviatrix_gateway** **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
```
Comment (inc1t3Ful) [inline CHANGELOG.md]: 3. Implemented cloud_type check to catch incorrect ``ha_subnet`` usage for **aviatrix_gateway**, **aviatrix_spoke_gateway**, and **aviatrix_transit_gateway**
4. Implemented ``ha_gw_size`` check to catch incorrect usage when enabling HA for **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**

**Pull request merged**